superset/dashboards/dao.py

```diff
@@ -265,6 +265,7 @@ def set_dash_metadata(  # pylint: disable=too-many-locals
         md["refresh_frequency"] = data.get("refresh_frequency", 0)
         md["color_scheme"] = data.get("color_scheme", "")
         md["label_colors"] = data.get("label_colors", {})
+        md["shared_label_colors"] = data.get("shared_label_colors", {})
```

Comment:
The `shared_label_colors` will be presented to the user in the advanced settings of the Dashboard. I think we should hide them in the frontend so the user never sees them if possible.

Reply:
Agreed.  As well as we need to filter out this field from the user input.